packages/node_modules/@node-red/editor-client/src/js/ui/view.js

```diff
@@ -2951,17 +3004,77 @@ RED.view = (function() {
                                 .attr("height",function(d){return d.h})
                                 .classed("red-ui-flow-node-highlighted",function(d) { return d.highlighted; })
                             ;
+                            var l = "";
+                            if (d._def.label) {
+                                l = d._def.label;
+                                try {
+                                    l = (typeof l === "function" ? l.call(d) : l)||"";
+                                    l = RED.text.bidi.enforceTextDirectionWithUCC(l);
+                                } catch(err) {
+                                    console.log("Definition error: "+d.type+".label",err);
+                                    l = d.type;
+                                }
+                            }
+                            var sa = convertLineBreakCharacter(l);
+                            var sn = sa.length;
+                            var st = "";
+                            var yp = d.h / 2 - (sn / 2) * 24 + 16
+                            if(labelLineNumber<sn) {
+                                for(var i=labelLineNumber;i<sn;i++) {
+                                    var text = node.append("svg:text")
+                                        .attr("class","red-ui-flow-node-label")
+                                        .attr("id","red-ui-flow-node-label-"+i)
```

Comment:
If I'm reading this right, every node add an element with an `id` of `red-ui-flow-node-label-0`. You must not have multiple elements with the same id.

Reply:
I fixed the id. https://github.com/node-red/node-red/pull/2242/commits/7957ec43692f00277f96ed23de509a7e841e86a5
Please check them.